components/Search/SearchForm.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+
+import TextField from 'material-ui/TextField';
+import RaisedButton from 'material-ui/RaisedButton';
+import ActionSearch from 'material-ui/svg-icons/action/search';
+
+class SearchForm extends Component {
+
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const { hintText, onSubmit, query } = this.props;
+
+    const onEnter = (e) => {
+      
+      if (e.keyCode !== 13)
```

Comment:
I think using onEnter to submit, rather than wrapping it in a form element and using onSubmit is a bit strange. 

Only time I think to use something like onEnter is if you're using textareas as text inputs, as enters are a carriage return